agent/04_agent_configure.sh

```diff
@@ -509,6 +509,15 @@ fi
 
 if [[ "${NUM_MASTERS}" > "1" ]]; then
    set_api_and_ingress_vip
+else
+  # For SNO clusters, at least the api dns entry must be set
+  # otherwise oc/openshift-install commands requiring the
+  # kubeconfig will not work properly
+  ip=${AGENT_NODES_IPS[0]}
+  if [[ "$IP_STACK" != "v4" ]]; then
```

Comment:
Has this been tested with dual stack? Right now we treat ipv4 as primary for that so I wonder if using the ipv6 address for the API DNS will work right.

Reply:
@bfournie was working on the dual-stack, not sure if he maybe tried out that configuration. Our immediate need anyhow is to have the CI job working, which currently it's just ipv4 based, so eventually we could followup on the SNO dualstack case later when it will be added to the CI

Reply:
Cool, that's good to know. Maybe ipv6-primary dual stack will work for regular IPI deployments too then.